index.html

```diff
@@ -40,7 +40,7 @@
 </head>
 <body>
   <section class="site-banner">
-    Get an intro to Grain at the&nbsp;<a href="https://webassembly-summit.org/" target="_blank" rel="noopener">2021 WebAssembly Summit</a>
+    Find out about our latest release<a href="https://grain-lang.org/blog/2021/04/29/new-release-grain-v0.3.0-barley/" target="_blank" rel="noopener"> on our blog</a>
```

Comment:
This may do the right thing:
```suggestion
    <span>Find out about our latest release<a href="https://grain-lang.org/blog/2021/04/29/new-release-grain-v0.3.0-barley/" target="_blank" rel="noopener"> on our blog</a></span>
```

Reply:
I was trying to hack on that a bit. Might also need to inject a `&nbsp;`

Reply:
Yeah, that's what we've been doing up until now. The website really needs some love 😢